templates/tab/js/m365/src/components/sample/AzureFunctions.jsx

```diff
@@ -1,13 +1,16 @@
-import React from "react";
+import { useContext } from "react";
 import { Button, Loader } from "@fluentui/react-northstar";
-import { useData } from "./lib/useData";
+import { useData } from "@microsoft/teamsfx-react";
 import { BearerTokenAuthProvider, createApiClient, TeamsFx } from "@microsoft/teamsfx";
+import { TeamsFxContext } from "../Context";
 
-var functionName = process.env.REACT_APP_FUNC_NAME || "myFunc";
+const functionName = process.env.REACT_APP_FUNC_NAME || "myFunc";
 
-async function callFunction() {
+async function callFunction(teamsfx?: TeamsFx) {
```

Comment:
```suggestion
async function callFunction(teamsfx) {
```
Type annotations can only be used in TypeScript files.
Then can remove `TeamsFx` from import.

Reply:
Thanks Kuojian! I've fixed the issue and remove the TeamsFx from import。